app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def flash_class(type)
```

Comment:
Is this module using bootstrap syntax to style?

Reply:
Yeah! https://getbootstrap.com/docs/4.1/components/alerts/

Reply:
This is interesting this might be the first time I've seen a case being used. Looks like this adds the class directly tag. Good work on this - very creative